### DISCUSSION TITLE
Nextjs host share routing with CRA micro-frontend

### DISCUSSION THREAD

**Discussion by SombreroElGringo** [module-federation/module-federation-examples#2939] [Q&A]
Hello,

I would like to know how we should share our nextjs `Link` and `useRouter` with a CRA micro-frontend? 

I saw 2 examples between a CRA host and CRA micro-frontend, but it seems more tricky with a nextjs host. Do you have any examples or suggestions to achieve this?

**Comment by ScriptedAlchemy** [comment]
https://github.com/module-federation/core/issues/3153
